### ISSUE TITLE
How to prevent recursion?

### ISSUE THREAD

**Issue by lukewis** [IvanBoyko/vsts-regex-replace#3]
How can I do a replacement on a single file?  The default seems to be recursively searching subdirectories.  How can I prevent this recursion?

**Comment by IvanBoyko** [comment]
Just put the specific file path in Files Mask.
If you use ** as in examples, this is what makes it recursive.

**Comment by lukewis** [comment]
Thanks for the response!  Unfortunately that didn't seem to work for me.  The files mask I tried was "package.json" (which lived in the root of my repository), but it also did replacements in other package.json files it found under the node_modules folder (the node_modules folder is also at the root of the repository).  The workaround I'm using right now is that I first copy package.json to "tmp/package.json" and then I set the files mask to "tmp/package.json".  Since there's no other directories under the "tmp" folder, there's no where to recurse, so this achieves my goal, but it feels pretty clunky.  Maybe a checkbox that indicates whether or not to recurse would solve the issue?

**Comment by IvanBoyko** [comment]
Yes, you are right - it's actually hard-coded to do `gci -Recurse`: https://github.com/IvanBoyko/vsts-regex-replace/blob/master/Tasks/RegexReplace/RegexReplace.ps1#L61
Let me think about it.

**Comment by IvanBoyko** [comment]
So... I don't mind making this improvement.
I've recovered the project (haven't touched it in almost a year), but the problem is that I can't recover my old publisher to push a new version of the extension to the VSTS Marketplace.
I've asked Microsoft support to help.
Let's see.
